ui/src/ducks/app/nodes.js

```diff
@@ -439,6 +474,39 @@ export function* stopRefreshNodes() {
   );
 }
 
+export function* fetchNodesIPsInterface() {
+  let result;
+  // Check if Salt API is already authenticated
+  // If not, wait for the CONNECT_SALT_API action.
+  const isSaltAPIAuthenticated = yield select(isSaltAPIAuthenticatedSelector);
+  if (isSaltAPIAuthenticated) {
+    result = yield call(ApiSalt.getNodesIPsInterfaces);
+  } else {
+    // eslint-disable-next-line no-unused-vars
+    const { success, failure, timeout } = yield race({
```

Comment:
It's sad that we are missing proper error-handling for this :confused: I'll open a debt ticket to track this.